ktor-utils/jvm/src/io/ktor/util/EncodersJvm.kt

```diff
@@ -48,11 +59,31 @@ private fun CoroutineScope.inflate(
         val header = source.readPacket(GZIP_HEADER_SIZE)
         val magic = header.readShortLittleEndian()
         val format = header.readByte()
-        val padding = header.readBytes()
+        val flags = header.readByte().toInt()
```

Comment:
Memory leak here: header should be consumed entirely or discarded

Reply:
Fixed in 6bdf82850036b0c532307429bc460c988263dddd.